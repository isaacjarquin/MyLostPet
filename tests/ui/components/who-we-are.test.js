import "react-native"
import React from "react"
import WhoWeAre from "../../../components/who-we-are"

import renderer from "react-test-renderer"

it("renders correctly", () => {
    const tree = renderer.create(
        <WhoWeAre />
    ).toJSON()
    expect(tree).toMatchSnapshot()
})
