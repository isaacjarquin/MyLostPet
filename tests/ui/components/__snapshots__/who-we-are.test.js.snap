// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "grey",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "height": "100%",
        "justifyContent": "flex-start",
        "width": "100%",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/images/tirma.jpg",
        }
      }
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": "100%",
            "width": "100%",
          },
          undefined,
        ]
      }
    />
    <RCTScrollView
      style={
        Object {
          "backgroundColor": "black",
          "opacity": 0.8,
        }
      }
    >
      <View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "alignSelf": "center",
              "backgroundColor": "transparent",
              "color": "white",
              "fontSize": 20,
              "fontWeight": "bold",
              "margin": 10,
              "marginBottom": 10,
            }
          }
        >
          ¿Quiénes somos?
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "margin": 10,
              "marginBottom": 10,
              "opacity": 0.8,
            }
          }
        >
          My lost pet es una organización sin fines de lucro compuesta por un grupo de voluntarios unidos por el amor hacia los animales y la convicción de que, juntos, podemos hacer mucho por ellos.
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "margin": 10,
              "marginBottom": 10,
              "opacity": 0.8,
            }
          }
        >
          Esta iniciativa fue creada con el objetivo de facilitar todo lo posible la búsqueda de nuestras mascotas cuando éstas se encuentran extraviadas.
        </Text>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
